<launch>
  <!-- Interface to the Arduino used for the low-level control -->
  <node name="vending_machine" pkg="rosserial_python" type="serial_node.py" args="/dev/arduino_yun" required="true">
    <!-- Parameter values are time in milliseconds -->
    <param name="trigger_time" value="500"/>
    <param name="dispensing_time" value="8000"/>
    <param name="dispensed_signal_time" value="1000"/>
    <remap from="arduino/drink_order" to="drink_order"/>
    <remap from="arduino/order_result" to="order_result"/>
  </node>
</launch>
